easybuild/easyconfigs/p/protobuf/protobuf-3.6.1.2_fix_TensorFlow_issue_26155.patch

```diff
@@ -0,0 +1,39 @@
+Fix problem with building protobuf with CUDA 10.1
```

Comment:
@akesandgren How is this patch tied to CUDA (since we're building `protobuf` with `GCCcore` here)?

Reply:
Ah, forgot that part. It's needed for TensorFlows 1.13.1 (with CUDA) use of protobuf.
But TF still doesn't use the external protobuf, just wanted to keep them in sync.

It's the include file map.h that is the culprit... so the comment should be building against protobuf...

Reply:
Better now?

Reply:
Sure, I can live with this.